apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -27,6 +28,7 @@ function SectionProgressSelector({
   progressTableV2ClosedBeta,
   sectionId,
   hasSeenProgressTableInvite,
+  isInV2Navigaton,
```

Comment:
```suggestion
  isInV1Navigaton = false,
```
Optional nit: if we end up keeping this, then having the v2 as the default will make logic deletion easier when we are able to delete v1 code.